specification/hugr.md

```diff
@@ -1253,8 +1257,10 @@ The new hugr is then derived as follows:
     copy of the `SrcNode` of $\sigma\_\mathrm{out}$ according to the specification $\sigma\_\mathrm{out}$.
     The target port must have an existing edge whose source is in $R$; this edge
     is removed.
-4.  For each $(n, t = T(n))$, append the copy of $n$ to the list
-    of children of $t$ (adding a hierachy edge from $t$ to $n$).
+4.  Let $N$ be the ordered list of the copies made in $\Gamma$ of the children of the root node of $H$.
```

Comment:
This is a bit more complex than append-to-end, but I think it may be necessary for replacing particular children of Conditionals (even without changing their number); CFG entry nodes; and Input/Output nodes. Generally you'll need to provide exactly one Input if-and-only-if you remove an Input, sort of thing, but it seems reasonable that you might???